src/presentation/pages/Home/HomePage.module.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../../app/styles/base/settings';
+
+.homePage {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	align-items: center;
+
+	&__item {
+		border: solid 1px $dark-grey;
+		margin-top: 9rem;
+		width: 70%;
+		height: 17rem;
+	}
+
+	&__text {
+		color: $mid-grey;
+		margin-top: 4rem;
+		width: 33rem;
+		height: 10rem;
+	}
+
+	&__latestSection {
+		display: flex;
+		align-items: center;
+		width: 100%;
+		height: 100%;
+		padding: 2rem;
+		background-color: $light-grey;
+
+		&Item {
+			border: solid 1px $dark-grey;
+			height: 13rem;
+		}
+
+		&Text {
+			padding: 4rem !important;
```

Comment:
Is there any reason for using `!important` here?

Reply:
I totally forgot to set linting. I will do this Right now. Thanks